app/styles/auk/_auk-additions.scss

```diff
@@ -400,3 +400,7 @@ table tr.lt-expanded-row {
     text-transform: none;
   };
 }
+
+.auk-alert-stack {
+  z-index: 10000;
+}
```

Comment:
the alert stack should already by quite high up the z-index scale.
https://github.com/kanselarij-vlaanderen/au-kaleidos-css/blob/6d763bddf5b94dc0c5d86dd11c9e8e1a8a2e5c43/_s-z-index.scss#L12

Reply:
It's because AuModals are set to z-index 9999, so the alerts don't show in the document preview modal. None of the alerts did, so any errors were invisible, but now it's quite obvious because (with z-index 1700) the toasts about sign flow creation where hidden until the user clicked off the modal.

Reply:
I see, this must be in the ember-appuniversum repo.